py3-cloudpickle.yaml

```diff
@@ -0,0 +1,41 @@
+# Generated from https://pypi.org/project/cloudpickle/
+package:
+  name: py3-cloudpickle
+  version: 2.2.1
+  epoch: 0
+  description: Extended pickling support for Python objects
+  copyright:
+    - license: BSD 3-Clause License
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      README: 'CONFIRM WITH: curl -L https://files.pythonhosted.org/packages/5f/51/913ecca3970a2227cf4d5e8937df52cc28f465ac442216110b8e3323262d/cloudpickle-2.2.1.tar.gz | sha256sum'
```

Comment:
I think we can remove all the `README: 'CONFIRM WITH: curl -L ...` lines in this PR, they're not needed

Reply:
Other than that I think it looks good!

Reply:
done! thank you!